tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```diff
@@ -134,6 +152,19 @@ spec:
           echo "${build}"
         }
 
+        create_auth_file() {
+          mkdir -p "${HOME}/.config/containers"
+          targetIndex="$(params.targetIndex)"
+          authName="${targetIndex%:*}"
+
+          # disabling debug to not leak the token
+          set +x
+          jq --arg authName "${authName}" \
+             --arg token "$(base64 -w 0 < <(printf %s "${TARGET_INDEX_CREDENTIAL}"))" \
+             '.auths[$authName].auth = $token' <<< '{}' > "${HOME}/.config/containers/auth.json"
```

Comment:
Instead of `<<< '{}'`, you can use `jq -n` or `jq --null-input`.

Reply:
I tried beforehand, but probably used it wrong. I will crosscheck.

Reply:
fixed.